Source/Library/GameBoardWindow/GameBoardController.cpp

```diff
@@ -0,0 +1,47 @@
+#include "TicTacToe/Library/GameBoardWindow/GameBoardController.hpp"
+
+using namespace boost::signals2;
+using namespace TicTacToe;
+
+GameBoardController::GameBoardController(int x_badge_count, int o_badge_count)
+    : m_x_badge_count(x_badge_count),
+      m_o_badge_count(o_badge_count),
+      m_game_state(GameModel::State::CONTINUE),
+      m_game_board_window(m_game_model.get_board(), Board::Token::X) {
+  init_game_board();
```

Comment:
Remove the ```init_game_board``` and ```show_game_board``` and inline those methods into the constructor. The point was to avoid inlining long lambda expressions that implement game logic directly in the constructor. It makes it very hard as a reader to follow the logic of the constructor when non-constructor logic is mixed in with it.